README.md

```diff
@@ -364,11 +423,16 @@ dictionary AIAssistantCreateOptions {
   AICreateMonitorCallback monitor;
 
   DOMString systemPrompt;
-  sequence<AIAssistantPrompt> initialPrompts;
+  sequence<AIAssistantInitialPrompt> initialPrompts;
   [EnforceRange] unsigned long topK;
   float temperature;
 };
 
+dictionary AIAssistantInitialPrompt {
+  AIAssistantInitialPromptRole role;
```

Comment:
Is this `role` compulsory or shall we give it a default value? If it's compulsory, do we reject the prompt request?

Reply:
Great catch. Both role and content should be required in Web IDL, so that argument validation will fail at the bindings layer if they're missing.